espflash/Cargo.toml

```diff
@@ -40,8 +40,8 @@ clap_complete = { version = "4.4.3", optional = true }
 comfy-table = { version = "7.0.1", optional = true }
 crossterm = { version = "0.25.0", optional = true } # 0.26.x causes issues on Windows
 ctrlc = { version = "3.4.0", optional = true }
-defmt-decoder = { version = "=0.3.8", features = ["unstable"], optional = true }
-defmt-parser = { version = "=0.3.3", features = ["unstable"], optional = true }
+defmt-decoder = { version = "0.3.9", features = ["unstable"], optional = true }
+defmt-parser = { version = "0.3.3", features = ["unstable"], optional = true }
```

Comment:
One reason why these were pinned was that defmt does not guarantee MSRV even for patch releases. I would recommend keeping the dependencies pinned, otherwise you may find that CI can break without changing anything defmt-related.

Reply:
Oh! Thanks for the heads up, just fixed it!